content/applications/finance/accounting/payables/supplier_bills/invoice_digitization.rst

```diff
@@ -16,14 +16,17 @@ created and populated based on the scanned invoices.
      <https://www.odoo.com/slides/slide/digitize-bills-with-ocr-1712>`_.
 
 .. note::
-   - The more documents you scan, the better the system identifies the correct data.
+   It is essential to review and correct (if needed) the information uploaded during digitization.
+   Then, you have to post the document by clicking on :guilabel:`Confirm`. In this manner, the AI
+   learns, and the system identifies the correct data in the future.
 
 Configuration
 =============
 
 In :menuselection:`Accounting --> Configuration --> Settings --> Digitalization`, check the box
 :guilabel:`Document Digitalization` and choose whether :guilabel:`Vendor Bills` and
```

Comment:
```suggestion
In :menuselection:`Accounting --> Configuration --> Settings --> Digitalization`, check the box
:guilabel:`Document Digitalization` and choose whether :guilabel:`Vendor Bills` and
```
On 15.0, it's still called Bill Digitalization, and not just Digitalization. Could you test everything on Odoo 15, since it's the version you are targeting with this doc?

Reply:
I tested on 15.3 I thought it was alright @jcs-odoo could you please confirm?